### ISSUE TITLE
BD SUP editor: Add new tool 'Move vertical position'

### ISSUE THREAD

**Issue by manuelrn** [SubtitleEdit/subtitleedit#7517]
Hello again!

I would like to know if it would be possible to implement a new option in the "Tools" menu ("Move vertical position", or something similar), I would be very interested and it would be very useful!

The idea would be a window with 2 text entries and 2 buttons.
The 2 text entries would be numbers (integer, no decimals), one integer would be for the top lines and the other integer would be for the bottom lines. The 2 values (top lines and bottom lines) can also be negative integers, not just positive.
And the 2 buttons would simply be "OK" and "Cancel".

This tool would modify the value of the horizontal position (Y) on all lines. To the Y value will be applied the sum of the integer value (so if the integer is positive the Y value will increase, and if the integer is negative then the Y value will decrease).
If the line is at the top it will be applied the integer that the user has entered in the top lines option, and if the line is at the bottom it will be applied the integer that the user has entered in the bottom lines option.
The bottom lines would be considered those whose Y value is greater than or equal to the vertical resolution of the PGS (for example 1080) by subtracting the height of the image from the line and then dividing the result by 2.
<br />

In summary, the operation to be performed on each line would be as follows:
if Y >= ((ScreenHeight - bmpHeight) / 2): Y = Y + bottomInteger
else: Y = Y + topInteger
<br />

This is quite useful in some cases, for example when you find some PGS subtitle that is positioned outside the image (inside the black bands) and you want to move it inside the image so you need to move it X amount of pixels.

Thank you very much and best regards! ;)

**Comment by eddilara** [comment]
why not just use .ass format
